core/src/main/java/org/polypheny/db/type/entity/PolyInterval.java

```diff
@@ -95,17 +149,30 @@ public Long getMonths() {
 
     @Override
     public Object toJava() {
-        return value;
+        return millis;
     }
 
 
-    public long getMillis() {
-        log.warn( "might adjust" );
-        return switch ( qualifier.getTimeUnitRange() ) {
-            case YEAR -> value.longValue() * 24 * 60 * 60 * 1000;
-            case MONTH -> value.longValue();
-            default -> throw new NotImplementedException( "since Epoch" );
-        };
+    public Long getMergedMillis() {
```

Comment:
Can we remove this method?

Reply:
Removed.

Reply:
You have removed getMillis() but not getMergedMillis()?

Reply:
Removed calls, but not method itself. Fixed